specification/liftrqumulo/resource-manager/readme.python.md

```diff
@@ -2,20 +2,17 @@
 
 These settings apply only when `--python` is specified on the command line.
 Please also specify `--python-sdks-folder=<path to the root directory of your azure-sdk-for-python clone>`.
-Use `--python-mode=update` if you already have a setup.py and just want to update the code itself.
-
 
 ``` yaml $(python)
-title: QumuloMgmtClient
 azure-arm: true
 license-header: MICROSOFT_MIT_NO_VERSION
-package-name: azure-mgmt-qumulo
-namespace: azure.mgmt.qumulo
+package-name: azure-mgmt-qumulostorage
+namespace: azure.mgmt.qumulostorage
```

Comment:
Python already released GA https://pypi.org/project/azure-mgmt-qumulo/1.0.0/ so it is not permitted to change the package name anymore.

Reply:
Sure